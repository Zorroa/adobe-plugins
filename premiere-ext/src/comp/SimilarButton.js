import React from 'react'

function SimilarButton(props) {
    
  const callback = props.onpress

  return (
    <div className="thumb-button" onClick={callback}>
        <svg viewBox="0 0 20 20" height="20" color="#ffffff">
            <path fill="currentColor" d="M12 0a8 8 0 013.293 15.293A8 8 0 114.708 4.707 8 8 0 0112 0zM4.014 7.516l-.106.096a6 6 0 108.577 8.374 8 8 0 01-8.47-8.47zM8 6a6 6 0 00-1.743.257 6 6 0 007.486 7.486A6 6 0 008 6zm4-4a5.985 5.985 0 00-4.485 2.014 8 8 0 018.47 8.47A6 6 0 0012 2z"></path>
        </svg>
  </div>
  )
}

export default SimilarButton
