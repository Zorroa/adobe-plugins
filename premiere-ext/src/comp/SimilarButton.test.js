import renderer from "react-test-renderer"
import SimilarButton from "./SimilarButton"

describe("<SimilarButton/>", () => {
  it("should render properly", () => {
    const wrapper = renderer.create(
        <SimilarButton onpress={()=>{}} />
    )
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
