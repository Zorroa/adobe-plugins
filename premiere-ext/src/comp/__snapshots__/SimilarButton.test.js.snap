// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimilarButton/> should render properly 1`] = `
<div
  className="thumb-button"
  onClick={[Function]}
>
  <svg
    color="#ffffff"
    height="20"
    viewBox="0 0 20 20"
  >
    <path
      d="M12 0a8 8 0 013.293 15.293A8 8 0 114.708 4.707 8 8 0 0112 0zM4.014 7.516l-.106.096a6 6 0 108.577 8.374 8 8 0 01-8.47-8.47zM8 6a6 6 0 00-1.743.257 6 6 0 007.486 7.486A6 6 0 008 6zm4-4a5.985 5.985 0 00-4.485 2.014 8 8 0 018.47 8.47A6 6 0 0012 2z"
      fill="currentColor"
    />
  </svg>
</div>
`;
